cmake/option/option_armhf-linux.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright 2015 Samsung Electronics Co., Ltd.
```

Comment:
Why 2015? Is this copied from somewhere else? IMHO should be `2015-2017` or `2017`